code/game/objects/items/robot/items/tools.dm

```diff
@@ -156,7 +156,7 @@
 			energy = clamp(energy + energy_recharge * seconds_per_tick, 0, maxenergy)
 			return
 	if(host.cell && (host.cell.charge >= (host.cell.maxcharge * cyborg_cell_critical_percentage)) && (energy < maxenergy))
-		host.cell.use(energy_recharge * seconds_per_tick * energy_recharge_cyborg_drain_coefficient)
+		host.cell.use(energy_recharge * seconds_per_tick * energy_recharge_cyborg_drain_coefficient KILO WATTS)
```

Comment:
You should scale the max energy and energy charge and apply units there instead.

Reply:
Looks like `energy_recharge_cyborg_drain_coefficient` is used in this one place so i'll replace its value over here.

Also i cannot scale `energy_recharge` because it is used for the borgs own internal charge variable(yes it uses 2 power sources for some reason)